steps/deploy-to-updraft/0.0.1/step.yml

```diff
@@ -0,0 +1,73 @@
+title: deploy-to-updraft
```

Comment:
We recommend to have a user readable step title as this will appear in the search panel on the website. I think in your case it is simple because the step identifier so for example just use `Deploy to Updraft`.

Reply:
Fixed